common/types/activation.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/hex"
 	"fmt"
 	"github.com/spacemeshos/go-spacemesh/common/util"
+	"github.com/spacemeshos/go-spacemesh/log"
```

Comment:
my only comment, you could've redefined the `Field` interface in `types` to prevent this import.